.appcast.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can't drop a review starting with an XML file on me like this.

Reply:
Sorry, buddy.